docs/feature_advanced_keycodes.md

```diff
@@ -134,6 +134,8 @@ For example, if you define a key as `OSM(MOD_LSFT)`, you can type a capital A ch
 
 One shot keys also work as normal modifiers. If you hold down a one shot key and type other keys, your one shot will be released immediately after you let go of the key.
 
+Additionally, for both One Shot keys, if you hit the key 5 times in a short period of time, it will lock that key.  Meaning that it will act as if you held the key down without having to actually hold the key down, keeping the modifier or layer active until you hit the key again.
```

Comment:
I think that first sentence can be better worded as 

"Additionally, hitting keys five times in a short period will lock that key. This applies for both One Shot keys."

Reply:
That's what I get for trying to write at 2am